docs/quick_start/installing.rst

```diff
@@ -58,12 +58,12 @@ Once complete you can add the repository itself into the apt configuration file
     
     .. code-block:: console
 
-       deb  http://archives.fledge-iot.org/latest/buster/armv7l/ /
+       deb  http://archives.fledge-iot.org/latest/bullseye/armv7l/ /
 
     to the end of the file.
 
     .. note:: 
-       Replace `buster` with  `stretch` or `bullseye` based on the OS image used.
+       Replace `bullseye` with  the name of the version of the Raspberry Operating System you have installed.
```

Comment:
we only support bullseye and buster!

We should have a list of supported platforms which is actually missing in our doc! We need to create a JIRA